### PR TITLE
Refresh Polish translation

### PR DESCRIPTION
**Pull request by fdmarcin**
Also, updated CC licence link.

### CODE REVIEW

**Comment by Xylakant** [comment]
ping @pestkownica

**Comment by fdmarcin** [comment]
Done, @hagenburger. I'm leaving the translation here.

File: source/localizable/_code-of-conduct.pl.md [hunk 1]
```diff
@@ -2,52 +2,52 @@ Cel
 ---
 
 Podstawowym celem wszelkich konferencji i grup użytkowników powołujących się na ten Kodeks postępowania jest otwartość na jak największą liczbę osób o jak najbardziej urozmaiconych i różnorodnych korzeniach. Z tego powodu zobowiązujemy się do zapewnienia życzliwej, bezpiecznej i przyjaznej atmosfery wszystkim, bez względu na płeć, orientację seksualną, sprawność, pochodzenie etniczne, status społeczno-ekonomiczny i religię (lub jej brak).
-
+https://creativecommons.org/licenses/by-sa/4.0/deed.pl
```
Comment (chastell) [inline source/localizable/_code-of-conduct.pl.md]: I think this link might’ve gotten mis-pasted here.

**Comment by chastell** [comment]
Thanks so much, @fdmarcin! The wording changes are great (I’m not sure about the extra commas, but they don’t bother me either). I’m only surprised by the extra link added in place of a paragraph break.

File: source/localizable/_code-of-conduct.pl.md (hunk 1 above)
Comment (fdmarcin) [reply source/localizable/_code-of-conduct.pl.md]: It totally was :) I'm blaming my middle mouse button.

Comment (fdmarcin) [reply source/localizable/_code-of-conduct.pl.md]: Removed the link.

**Comment by chastell** [comment]
@fdmarcin Looks good, thanks!

@hagenburger Should I merge this or would you like to wait for another review first?

**Comment by hagenburger** [comment]
@chastell thanks for your feedback!

@1000miles @lauralindal @skade you recently discussed about processes when I wasn’t there. Did you set any rules on language reviews?

(Side node for all: The merge should use “Squash and merge”)

**Comment by chastell** [comment]
@hagenburger @1000miles @lauralindal @skade Argh, right, this is probably not the place to discuss this, but if there are clear rules what is mergeable by people like me (and when) I’d appreciate it a lot. ♥

(I merged #154 on a whim yesterday but maybe I shouldn’t have? I’m also not sure who has the power to (re)deploy the site; given that PR is not reflected in production I assume it’s not done automatically.)

**Comment by fdmarcin** [comment]
Hi, has anything changed?

**Comment by fdmarcin** [comment]
Hi, should anything be done with this, or can it be merged?
cc @hagenburger @chastell

**Comment by hagenburger** [comment]
@fdmarcin if @chastell is fine with it, we can merge. Could you maybe rebase to master and solve the merge conflicts?

**Comment by fdmarcin** [comment]
@hagenburger Is it okay now? I haven't done this before across forks instead of branches.

**Comment by hagenburger** [comment]
I merged that into master manually. Thank you!

**Pull request closed without merging**